binary-transcoders/src/main/kotlin/com/kamelia/sprinkler/codec/binary/encoder/composer/ComposedEncoder.kt

```diff
@@ -0,0 +1,78 @@
+@file:JvmName("ComposedEncoderFactory")
+
+package com.kamelia.sprinkler.codec.binary.encoder.composer
+
+import com.kamelia.sprinkler.codec.binary.encoder.UTF8StringEncoder
+import com.kamelia.sprinkler.codec.binary.encoder.core.Encoder
+import com.kamelia.sprinkler.codec.binary.encoder.core.EncoderOutput
+import java.nio.ByteOrder
+
+/**
+ * Creates a new encoder of type [T] using the given lambda [block]. The [block] parameter is a lambda accepting an
```

Comment:
I dunno about using "lambda" to refer to functions in general here.

this is a suggestion but :

```suggestion
 * Creates a new encoder of type [T] using the given function [block]. The [block] parameter is a function accepting an
```